Cargo.toml

```diff
@@ -0,0 +1,95 @@
+[package]
+name = "zinc"
+version = "0.1.0"
+authors = ["Zinc Developers <zinc@github.com>"]
+build = "build.rs"
+
+[lib]
+name = "zinc"
+crate-type = ["lib"]
```

Comment:
Actually, that's a bug in cargo with trying to link against syntax extensions while cross compiling for a target that doesn't support dylibs.

See rust-lang/cargo#1612. It was fixed by rust-lang/cargo#1617 but something is wrong with the PR on windows right now.